gustaf/utils/connec.py

```diff
@@ -238,10 +235,7 @@ def range_to_edges(range_, closed=False, continuous=True):
 
     # continuous edges
     indices = np.repeat(indices, 2)
-    if closed:
-        indices = np.append(indices, indices[0])[1:]
-    else:
-        indices = indices[1:-1]
+    indices = np.append(indices, indices[0])[1:] if closed else indices[1:-1]
```

Comment:
Here the same.

Reply:
a bit weird but I am okay with leaving formatter to do its job

Reply:
also this seems like a bit of over kill - maybe next PR